### DISCUSSION TITLE
bs problem

### DISCUSSION THREAD

**Discussion by whoiscamm** [RareDevs/Rare#323] [General]
@loathingKernel u think we could take this to a call to make it easier for the both of us?

**Comment by loathingKernel** [comment]
No, because I am not tech support. As a project run by volunteers we do not have that. If you would like to explain the issue in depth I would be more than willing to help you if I can.

**Comment by whoiscamm** [comment]
no disrespect to u but man i give up

**Comment by loathingKernel** [comment]
There is no disrespect in that, you can do what you want with your time.

**Comment by whoiscamm** [comment]
[frozen_application_license.txt](https://github.com/RareDevs/Rare/files/13544790/frozen_application_license.txt)
what is this

**Comment by loathingKernel** [comment]
This is the license of cx_freeze, the method we use to package Python programs for distribution on windows.
